fhd_utils/modified_astro/apply_astrometry.pro

```diff
@@ -39,13 +39,16 @@ IF (Keyword_Set(xy2ad) AND Keyword_Set(ad2xy)) THEN $
 IF size(obs,/type) EQ 8 THEN BEGIN 
     astr=obs.astr
     JDate=obs.JD0
-ENDIF ELSE ignore_refraction=1
+ENDIF ELSE refraction=0
+
+if Keyword_Set(refraction) THEN refraction=1 ELSE refraction=0
```

Comment:
Minor point, but this is unnecessary since you only use `IF Keyword_Set(refraction)` below.